src/commandHandlers/help.ts

```diff
@@ -19,15 +19,9 @@ export const command = async (arg: [string, string], embed: MessageEmbed) => {
       false
     );
   } else {
-    let text = '';
-    commandList.forEach((commandItem) => {
-      text = text
-        .concat(`**${COMMAND_PREFIX}${commandItem.triggers[0]}**  `)
-        .concat(`${commandItem.description}\n`)
-        .concat(
-          'Usage:' + '`' + `${COMMAND_PREFIX}${commandItem.usage}\n` + '`'
-        );
-    });
+    const format = (commandItem: any) =>
```

Comment:
You can use the [CommandHandler](https://github.com/EddieJaoudeCommunity/EddieBot/blob/301c0fcedbdea8772f827f02263a8e59812f8704/src/commandHandlers/index.ts#L31) type for this instead of `any` (@RaisinTen created it a while back 😃) :
```suggestion
    const format = (commandItem: CommandHandler) =>
```

Reply:
I didn't realise we already had a command typing. 😅

Reply:
Good to see others working on this! :slightly_smiling_face: